### PR TITLE
configlet: AddRack: Handle the possibility of pfcwd not running

### PR DESCRIPTION
**Pull request by renukamanavalan**
### Description of PR

Summary:
Fixes # 4171

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### How did you do it?
Some lab devices could have pfcwd disabled intentionally. Handle it.

#### How did you verify/test it?
This issue does not occur again

### CODE REVIEW

File: tests/configlet/util/configlet.py [hunk 1]
```diff
@@ -18,10 +18,13 @@ def get_pfc_time():
     ret = 0
     pfc_wd = config_db_data_wo_t0.get("PFC_WD", {})
     for n, val in pfc_wd.items():
-        ret = int(val["detection_time"])
+        ret = int(val.get("detection_time", 0))
+        if ret:
```
Comment (neethajohn) [inline tests/configlet/util/configlet.py:22]: if not ret?

Comment (renukamanavalan) [reply tests/configlet/util/configlet.py:22]: This function will return 0; The PFC_WD entry for this port will not be added, if 0.

```
        if pfc_time:
            # "PFC_WD"
            pfc_wd[local_port] = {
                    "action": "drop",
                    "detection_time": pfc_time,
                    "restoration_time": pfc_time }
```

**Comment by renukamanavalan** [comment]
Build failure:
2021-09-06T16:45:57.4853588Z ==================================== ERRORS ====================================
2021-09-06T16:45:57.4855142Z _________________ ERROR at setup of test_monit_status[vlab-01] _________________

**Review by neethajohn — APPROVED** [review]

**Pull request merged**